lcu_integration.py

```diff
@@ -40,14 +40,23 @@ def __init__(self):
         self._url = None
         self.install_directory = None
 
-    def connect_to_lcu(self):
+    def connect_to_lcu(self) -> bool:
+        logger.info("Waiting for the League client (5m timeout)")
         lcu_process = _get_lcu_process()
+
+        timeout = 0
```

Comment:
nit: I like the idea of a 5 minute timeout, but unfortunately this will not be a 5 minute timeout, just due to how it's implemented!

A few options:
- Update `timeout` to record the start time of the block and do a delta vs the current time at evaluation time to check
- Update `5m timeout` to `~5m timeout` just so it's clear it's not actually 5 minutes
- Ignore this comment since this is super nit-picky and I just wanted to point this out, it's not a major blocker in any way